docs/dashboard-setup/whitelisting.mdx

```diff
@@ -0,0 +1,96 @@
+
+Type in the URL you wish to whitelist in the text box and click on the `Add` button. The URL will be added to the list of whitelisted URLs. You can
+add as many URLs as you want.
+
+<div style={{ flexBasis: "300px", flexGrow: "1", textAlign: "center" }}>
+  <img src={DashboardImage} style={{ alignSelf: "center", maxWidth: "100%" }} alt="Dashboard Whitelist" />
+</div>
+
+### Whitelisting for Mobile apps
+
+You can use the Developer Dashboard to whitelist URLs for mobile apps as well. For Android, you can whitelist the URL by adding the `intent-filter`
+making the links universal. For iOS, you can whitelist the URL by adding the `Associated Domains` in the `Capabilities` section of the project. For an
+extensive guide on deep linking and app linking for Android apps, please refer to the following
+[blog](https://dev.to/shahbaz17/a-comprehensive-guide-to-deep-links-and-app-links-in-android-4n23).
+
+### Manual Whitelisting for Advanced Cases
+
+You can use the manual origin whitelisting method to whitelist URLs without using the Developer Dashboard. `OpenLoginAdapter` accepts a parameter
+called `originData` inside the `adapterSettings`. `originData` is a key value pair where the key is the origin URL and the value is a `signature`. The
+signature is generated by `whitelistUrl` function from the `@toruslabs/openlogin` package. The `whitelistUrl` function accepts the `clientId`,
+`clientSecret` and `origin` as parameters.
+
+:::warning Note
+
+Please perform this in a highly secure environment. The `clientSecret` should not be exposed to the public making this a risky process.
+
+:::
+
+```js
+import { OpenloginAdapter } from "@web3auth/openlogin-adapter";
+import { whitelistUrl } from "@toruslabs/openlogin";
+import { CommonPrivateKeyProvider } from "@web3auth/base-provider";
+
+const clientId = "YOUR_CLIENT_ID"; // get from https://dashboard.web3auth.io
+const clientSecret = "CORRESPONDING_CLIENT_SECRET"; // get from https://dashboard.web3auth.io
+const origin = "https://example.com";
+
+const privateKeyProvider = new CommonPrivateKeyProvider({ config: { chainConfig } });
+const sig = await whitelistUrl(clientId, clientSecret, origin);
+
+const openloginAdapter = new OpenloginAdapter({
+  privateKeyProvider,
+  adapterSettings: {
+    originData: { origin: sig },
+  },
+});
+```
+
+A snippet from the SDK reference for `originData` is given below:
+
+```ts
+    /**
+     * originData is used to verify the origin of dapp by iframe.
+     *
+     * @internal
+     * @remarks
+     * You don't have to pass originData explicitly if you have registered your dapp at
+     * {@link "https://dashboard.web3auth.io"| developer dashboard}.
+     *
+     * originData contains a signature of dapp's origin url which is generated using
+     * project's secret.
+     */
+    originData?: OriginData;
+```
+
+### How to secure deep linking via whitelisting strategies to avoid phishing attacks?
```

Comment:
Can you add some Security points from the [blog](https://dev.to/shahbaz17/a-comprehensive-guide-to-deep-links-and-app-links-in-android-4n23)?

`Security Concerns with Deep Links:` section